### PR TITLE
Update in.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**
https://www.angeltv.org/index.php https://www.angeltv.org/Contact.php

### CODE REVIEW

File: streams/in.m3u [hunk 1]
```diff
@@ -17,6 +17,30 @@ http://akaram.zecast.net/akaram-live/akaramkidz/index.m3u8
 https://d39iawgzv3h0yo.cloudfront.net/out/v1/1ef4344a3b4a41908915d58ac7bd5e23/index.m3u8
 #EXTINF:-1 tvg-id="AmritaTV.in" status="online",Amrita TV (576p)
 http://103.199.161.254/Content/amrita/Live/Channel(Amrita)/index.m3u8
+#EXTINF:-1 tvg-id="" status="online",Angel TV Africa (720p)
+https://cdn3.wowza.com/5/TDJ0aWNkNXFxWWta/angeltvcloud/ngrp:angelafrica_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Angel TV America (720p)
```
Comment (freearhey) [inline streams/in.m3u:22]:
```suggestion
#EXTINF:-1 tvg-id="AngelTVAmerica.in" status="online",Angel TV America (720p)
```
https://iptv-org.github.io/channel?id=AngelTVAmerica.in

File: streams/in.m3u [hunk 2]
```diff
@@ -17,6 +17,30 @@ http://akaram.zecast.net/akaram-live/akaramkidz/index.m3u8
...
+https://cdn3.wowza.com/5/TDJ0aWNkNXFxWWta/angeltvcloud/ngrp:angelamerica_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Angel TV Arabia (720p)
```
Comment (freearhey) [inline streams/in.m3u:24]:
```suggestion
#EXTINF:-1 tvg-id="AngelTVArabia.in" status="online",Angel TV Arabia (720p)
```
https://iptv-org.github.io/channel?id=AngelTVArabia.in

File: streams/in.m3u [hunk 3]
```diff
@@ -17,6 +17,30 @@ http://akaram.zecast.net/akaram-live/akaramkidz/index.m3u8
...
+https://cdn3.wowza.com/5/TDJ0aWNkNXFxWWta/angeltvcloud/ngrp:angelarabia_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Angel TV Australia (720p)
```
Comment (freearhey) [inline streams/in.m3u:26]:
```suggestion
#EXTINF:-1 tvg-id="AngelTVAustralia.in" status="online",Angel TV Australia (720p)
```
https://iptv-org.github.io/channel?id=AngelTVAustralia.in

File: streams/in.m3u [hunk 4]
```diff
@@ -17,6 +17,30 @@ http://akaram.zecast.net/akaram-live/akaramkidz/index.m3u8
...
+https://cdn3.wowza.com/5/TDJ0aWNkNXFxWWta/angeltvcloud/ngrp:angelaustralia_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Angel TV Chinese (720p)
```
Comment (freearhey) [inline streams/in.m3u:28]:
```suggestion
#EXTINF:-1 tvg-id="AngelTVChinese.in" status="online",Angel TV Chinese (720p)
```
https://iptv-org.github.io/channel?id=AngelTVChinese.in

File: streams/in.m3u [hunk 5]
```diff
@@ -17,6 +17,30 @@ http://akaram.zecast.net/akaram-live/akaramkidz/index.m3u8
...
+https://cdn3.wowza.com/5/TDJ0aWNkNXFxWWta/angeltvcloud/ngrp:angelchinese_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Angel TV Europe (720p)
```
Comment (freearhey) [inline streams/in.m3u:30]:
```suggestion
#EXTINF:-1 tvg-id="AngelTVEurope.in" status="online",Angel TV Europe (720p)
```
https://iptv-org.github.io/channel?id=AngelTVEurope.in

File: streams/in.m3u [hunk 6]
```diff
@@ -17,6 +17,30 @@ http://akaram.zecast.net/akaram-live/akaramkidz/index.m3u8
...
+https://cdn3.wowza.com/5/TDJ0aWNkNXFxWWta/angeltvcloud/ngrp:angeleurope_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Angel TV Far East (720p)
```
Comment (freearhey) [inline streams/in.m3u:32]:
```suggestion
#EXTINF:-1 tvg-id="AngelTVFarEast.in" status="online",Angel TV Far East (720p)
```
https://iptv-org.github.io/channel?id=AngelTVFarEast.in

File: streams/in.m3u [hunk 7]
```diff
@@ -17,6 +17,30 @@ http://akaram.zecast.net/akaram-live/akaramkidz/index.m3u8
...
+https://cdn3.wowza.com/5/TDJ0aWNkNXFxWWta/angeltvcloud/ngrp:angelfareast_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Angel TV Hebrew (720p)
```
Comment (freearhey) [inline streams/in.m3u:34]:
```suggestion
#EXTINF:-1 tvg-id="AngelTVHebrew.in" status="online",Angel TV Hebrew (720p)
```
https://iptv-org.github.io/channel?id=AngelTVHebrew.in

File: streams/in.m3u [hunk 8]
```diff
@@ -17,6 +17,30 @@ http://akaram.zecast.net/akaram-live/akaramkidz/index.m3u8
...
+https://cdn3.wowza.com/5/TDJ0aWNkNXFxWWta/angeltvcloud/ngrp:angelhebrew_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Angel TV India (720p)
```
Comment (freearhey) [inline streams/in.m3u:36]:
```suggestion
#EXTINF:-1 tvg-id="AngelTVIndia.in" status="online",Angel TV India (720p)
```
https://iptv-org.github.io/channel?id=AngelTVIndia.in

File: streams/in.m3u [hunk 9]
```diff
@@ -17,6 +17,30 @@ http://akaram.zecast.net/akaram-live/akaramkidz/index.m3u8
...
+https://cdn3.wowza.com/5/TDJ0aWNkNXFxWWta/angeltvcloud/ngrp:angelindia_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Angel TV Portuguese (720p)
```
Comment (freearhey) [inline streams/in.m3u:38]:
```suggestion
#EXTINF:-1 tvg-id="AngelTVPortuguese.in" status="online",Angel TV Portuguese (720p)
```
https://iptv-org.github.io/channel?id=AngelTVPortuguese.in

File: streams/in.m3u [hunk 10]
```diff
@@ -17,6 +17,30 @@ http://akaram.zecast.net/akaram-live/akaramkidz/index.m3u8
...
+https://cdn3.wowza.com/5/TDJ0aWNkNXFxWWta/angeltvcloud/ngrp:angelportuguese_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Angel TV Russian (720p)
```
Comment (freearhey) [inline streams/in.m3u:40]:
```suggestion
#EXTINF:-1 tvg-id="AngelTVRussian.in" status="online",Angel TV Russian (720p)
```
https://iptv-org.github.io/channel?id=AngelTVRussian.in

File: streams/in.m3u [hunk 11]
```diff
@@ -17,6 +17,30 @@ http://akaram.zecast.net/akaram-live/akaramkidz/index.m3u8
...
+https://cdn3.wowza.com/5/TDJ0aWNkNXFxWWta/angeltvcloud/ngrp:angelrussia_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Angel TV Spanish (720p)
```
Comment (freearhey) [inline streams/in.m3u:42]:
```suggestion
#EXTINF:-1 tvg-id="AngelTVSpanish.in" status="online",Angel TV Spanish (720p)
```
https://iptv-org.github.io/channel?id=AngelTVSpanish.in

**Review by freearhey — CHANGES_REQUESTED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**